### DISCUSSION TITLE
Looking for a release that includes type to useControl export

### DISCUSSION THREAD

**Discussion by sreeramsama** [uber/react-map-gl#2156] [General]
When is the commit (https://github.com/visgl/react-map-gl/commit/2845d6033c57f9d0166ba106d1a65cce260f65f1) going to be released?
We are seeing eslint errors for rule [@typescript-eslint/no-unsafe-call](https://typescript-eslint.io/rules/no-unsafe-call) as `useControl` returns type `any` as of the latest release 7.0.21.

**Comment by Pessimistress** [comment]
Just released 7.0.22. Sorry for the delay!

**Reply by sreeramsama** [reply]
Thank you!
